vowpalwabbit/parse_args.cc

```diff
@@ -445,7 +445,9 @@ input_options parse_source(vw& all, options_i& options)
                      "A^B^C. Note: this will become the default in a future version, so enabling this option will "
                      "migrate you to the new behavior and silence the warning."))
       .add(make_option("flatbuffer", parsed_options.flatbuffer)
-               .help("data file will be interpreted as a flatbuffer file"));
+               .help("data file will be interpreted as a flatbuffer file"))
+      .add(make_option("external_parser", parsed_options.external_parser)
```

Comment:
If all this is being extended at compile time, might as well have a cmake flag to enable this forked binary mode and hide this code behind that flag. Sort of like the flatbuffers being compiled out, this would be external parser compiled out.

Reply:
good point, done